Devscord.DiscordFramework/Middlewares/Contexts/ChangedPermissions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using Devscord.DiscordFramework.Commons;
+
+namespace Devscord.DiscordFramework.Middlewares.Contexts
```

Comment:
czemu to jest w Contexts?

Reply:
W sumie to nie wiedziałem gdzie dokładnie umieścić